test/relay.test.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+
+var chai = require('chai')
+  , expect = chai.expect
+  , resolver = require('../src/resolver')
+  , helper = require('./helper')
+  , sequelize = helper.sequelize
+  , Sequelize = require('sequelize')
+  , Promise = helper.Promise
+  , attributeFields = require('../src/attributeFields');
+
+import {
+  GraphQLString,
+  GraphQLInt,
+  GraphQLFloat,
+  GraphQLNonNull,
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLSchema,
+  graphql
+} from 'graphql';
+
+import {
+  sequelizeNodeInterface
+} from '../src/relay';
+
+import {
+  globalIdField,
+  connectionDefinitions,
+  connectionArgs,
+  connectionFromArray
+} from 'graphql-relay';
+
+function generateTask(id) {
+  return {
+    id: id,
+    name: Math.random().toString()
+  }
+}
+
+describe('relay', function () {
+  var User
+    , Task
+    , userType
+    , taskType
+    , taskConnection
+    , nodeInterface
+    , Project
+    , projectType
+    , userConnection
+    , nodeField
+    , schema;
+
+  before(function () {
+    sequelize.modelManager.models = [];
+    sequelize.models = {};
+    User = sequelize.define('user', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    Task = sequelize.define('task', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    Project = sequelize.define('name', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    User.Tasks = User.hasMany(Task, {as: 'tasks'});
+    Project.Users = Project.hasMany(User, {as: 'users'});
+
+    var node = sequelizeNodeInterface(sequelize, {Project: projectType, User: userType, Task: taskType});
```

Comment:
`nodeInterface`s would be using those types by reference to return them when querying for an item by its `id`. Since it doesn't actually call them until at all (only returns the reference) it does not matter. We've verified these tests all pass as well.